build.gradle

```diff
@@ -62,10 +58,6 @@ jar {
 javadoc {
   options.encoding = 'UTF-8'
   options.charSet = 'UTF-8'
-  options.links(
-          'http://docs.guava-libraries.googlecode.com/git-history/v18.0/javadoc/',
-          'https://docs.oracle.com/javase/8/docs/api/'
```

Comment:
So wait, why are these being removed? If anything, should be trying to link up to the SpongeAPI javadocs as well since those links can be resolved.

Reply:
I removed the guave ones because guava isn't used by the project and I think I accidentally removed the java ones by accident.